prow/scripts/governance.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+readonly MILV_IMAGE="magicmatatjahu/milv:0.0.6"
```

Comment:
You need to `source` `library.sh` file for `docker-in-docker` environment end execute `init`.